dataland-frontend/tests/component/components/resources/frameworkDataSearch/DatasetStatusIndicator.cy.ts

```diff
@@ -0,0 +1,69 @@
+import DatasetStatusIndicator from "@/components/resources/frameworkDataSearch/DatasetStatusIndicator.vue";
+import { DataMetaInformation } from "@clients/backend";
+import { DataTypeEnum, QAStatus } from "@clients/backend";
+describe("Component Tests for DatasetStatusIndicator", () => {
+  const acceptedAndActiveDataset: DataMetaInformation = {
+    dataId: "mock-data-id",
+    companyId: "mock-company-id",
+    dataType: DataTypeEnum.Lksg,
+    qaStatus: QAStatus.Accepted,
+    currentlyActive: true,
+    reportingPeriod: "mock-reporting-period",
+    uploaderUserId: "mock-uploader-user-id",
+    uploadTime: 1672527600000, // 1.1.2023 00:00:00:0000
+  };
+
+  it("Should not display anything if the dataset is active", () => {
+    cy.mountWithPlugins(DatasetStatusIndicator, {}).then((mounted) => {
+      void mounted.wrapper.setProps({
+        displayedDataset: acceptedAndActiveDataset,
+      });
+    });
+
+    cy.get("div[data-test=datasetStatusContainer]").should("not.exist");
+    cy.get("a[data-test=datasetStatusLinkToActive]").should("not.exist");
+  });
+
+  it("Should display a outdated warning message when the dataset is outdated", () => {
+    const outdatedDataset = structuredClone(acceptedAndActiveDataset) as DataMetaInformation;
+    outdatedDataset.currentlyActive = false;
+    outdatedDataset.qaStatus = QAStatus.Accepted;
+
+    cy.mountWithPlugins(DatasetStatusIndicator, {}).then((mounted) => {
+      void mounted.wrapper.setProps({
+        displayedDataset: outdatedDataset,
+      });
+    });
+
+    cy.get("div[data-test=datasetStatusContainer]").contains("outdated").should("exist");
+    cy.get("a[data-test=datasetStatusLinkToActive]").should("not.exist");
+  });
+  it("Should display a QA-Pending warning message when the dataset is pending QA", () => {
```

Comment:
The banner message with QA-pending can only appear for uploaders viewing there own uploads that are currently not yet approved, correct?

Reply:
Thats the idea (and of course admins can view them too)